### PR TITLE
replace `Begin` words in coordinated transactions with `Use`

### PR DESCRIPTION
**Pull request by SaitTalhaNisanci**
This PR replaces the action words in coordinated transactions. When I read the method `BeginOrContinueCoordinatedTransaction` for a very long time I assumed that It actually begins a coordinated transaction. Like it sends the `BEGIN` command. If it was sending a `BEGIN` command, I would name it just like this, so I find it confusing. Instead we do set up some state for later use. I have replaced `Begin` with `Use`.  Let me know if you have any other suggestion.

### CODE REVIEW

File: src/backend/distributed/transaction/transaction_management.c [hunk 1]
```diff
@@ -170,11 +170,12 @@ InitializeTransactionManagement(void)
...
 /*
- * BeginCoordinatedTransaction begins a coordinated transaction. No
- * pre-existing coordinated transaction may be in progress./
+ * UseCoordinatedTransaction sets up the necessary variables to use
+ * a coordinated transaction. No pre-existing coordinated
+ * transaction may be in progress.
  */
 static void
-BeginCoordinatedTransaction(void)
+UseCoordinatedTransaction(void)
```
Comment (marcocitus) [inline src/backend/distributed/transaction/transaction_management.c]: Do we really need this function?

File: src/backend/distributed/transaction/transaction_management.c [hunk 2]
```diff
@@ -111,18 +111,18 @@ static bool MaybeExecutingUDF(void);
...
 
 /*
- * BeginOrContinueCoordinatedTransaction starts a coordinated transaction,
- * unless one already is in progress.
+ * UseOrKeepCoordinatedTransaction sets up the necessary variables to use
+ * a coordinated transaction, unless one is already in progress.
  */
 void
-BeginOrContinueCoordinatedTransaction(void)
+UseOrKeepCoordinatedTransaction(void)
```
Comment (marcocitus) [inline src/backend/distributed/transaction/transaction_management.c]: If we remove the other function, could we omit "OrKeep"? Use seems descriptive enough.

Comment (SaitTalhaNisanci) [reply src/backend/distributed/transaction/transaction_management.c]: Yeah I will remove the other function and remove `OrKeep`

**Comment by SaitTalhaNisanci** [comment]
I will wait for @onderkalaci approval as well for this one.

**Pull request merged**